.kokoro/docker/Dockerfile

```diff
@@ -174,4 +174,11 @@ RUN useradd -d /h -u ${UID} ${USERNAME}
 # Allow nopasswd sudo
 RUN echo "${USERNAME} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
+# Setup DRIFT region tag parser
+ENV REGION_TAG_PARSER_DIR /region-tag-parser
+ENV PARSER_PATH /region-tag-parser/wizard-py/cli.py
```

Comment:
Can you do `ENV PARSER_PATH ${REGION_TAG_PARSER_DIR}/wizard-py/cli.py`?

Reply:
I thought about doing that myself. 🙂 

Done.